src/scss/components/accordion/_stepper.scss

```diff
@@ -99,4 +100,8 @@ hx-accordion.beta-hxStepper {
       margin-left: 0.5rem;
     }
   }
+
+  .beta-hxContainerStyle {
```

Comment:
Try moving this style into a `<div>` block:

```scss
div {
...
```

We may have to tweak later.

Reply:
Changes have been done, please review.